build.gradle

```diff
@@ -89,6 +89,12 @@ dependencies {
     // Image processing and OCR
     implementation 'net.sourceforge.tess4j:tess4j:5.12.0'
     implementation 'org.imgscalr:imgscalr-lib:4.2'
+
+    // Agolia Search (For Wiki)
+    // We should eventually switch to v4, but currently it doesn't serialize POJOs correctly...
```

Comment:
Is this tracked somewhere on their issue tracker? If so can we get a link here

Reply:
Figured out how to use v4.